tests/x11/toolkits/fltk.pm

```diff
@@ -21,8 +21,9 @@ sub run {
     script_run './fltk', 0;
     assert_screen [qw(ui-toolkit-fltk ui-toolkit-fltk-nomsg-display)];
     if (match_has_tag 'ui-toolkit-fltk-nomsg-display') {
-        wait_screen_change { send_key 'alt-f4' };
-        script_run './fltk', 0;
+        # Use desktop runner to refresh screen
```

Comment:
Looks great! can you please add `record_info` or `record_softfail` here the reason why you use this workaround?

btw, please run more VRs [like +50 iterations]

Reply:
Rerun 50 times.  All of them can pass: https://openqa.suse.de/tests/15710657#next_previous